cypress/pages/features/node-creator.ts

```diff
@@ -0,0 +1,57 @@
+import { BasePage } from "../base";
+import { INodeTypeDescription } from '../../packages/workflow';
+
+export class NodeCreator extends BasePage {
+	url = '/workflow/new';
+	getters = {
+		plusButton: () => cy.getByTestId('node-creator-plus-button'),
+		canvasAddButton: () => cy.getByTestId('canvas-add-button'),
+		searchBar: () => cy.getByTestId('search-bar'),
+		getCreatorItem: (label: string) => this.getters.creatorItem().contains(label).parents('[data-test-id="item-iterator-item"]'),
+		getNthCreatorItem: (n: number) => this.getters.creatorItem().eq(n),
+		nodeCreator: () => cy.getByTestId('node-creator'),
+		nodeCreatorTabs: () => cy.getByTestId('node-creator-type-selector'),
+		selectedTab: () => this.getters.nodeCreatorTabs().find('.is-active'),
+		categorizedItems: () => cy.getByTestId('categorized-items'),
+		creatorItem: () => cy.getByTestId('item-iterator-item'),
+		communityNodeTooltip: () => cy.getByTestId('node-item-community-tooltip'),
+		noResults: () => cy.getByTestId('categorized-no-results'),
+		activeSubcategory: () => cy.getByTestId('categorized-items-subcategory'),
+		expandedCategories: () => this.getters.creatorItem().find('>div').filter('.active').invoke('text'),
+	};
+	actions = {
+		openNodeCreator: () => {
+			this.getters.plusButton().click();
+			this.getters.nodeCreator().should('be.visible')
+		},
+			this.getters.getNthCreatorItem(n).click();
+		},
```

Comment:
This misresolved conflict prevents running the tests:
<img width="1437" alt="image" src="https://user-images.githubusercontent.com/44588767/203985012-3ad24a46-7706-43a1-9420-9338317c25a5.png">

Reply:
My bad, fixed